packages/app/documents/service.ts

```diff
@@ -725,3 +855,15 @@ function getWorkflowEdgesMatchingSourceAndTarget(
         edge => edge.source === source && edge.target === target
     )
 }
+
+function jsonPatch(
```

Comment:
Wrapping immutable-json-patch to use the errordata pattern. If any failures happening during the JSON Patch, want them converted into a "friendly" error